### ISSUE TITLE
[Documentation / FAQ]

### ISSUE THREAD

**Issue by rubyFeedback** [foxcpp/maddy#298]
Hey guys,

I'd love to explore alternatives to e. g. sendmail and all the other horrible tools out there
in regards to sending emails (I am not kidding; they are such a nightmare, also to setup
properly). So anyone who tries to better this, already deserves a medal.

There is one problem (for me) and this is ... how to work with maddy?

Would it be possible to add a FAQ, and, even more importantly, some things to do
in a post-installation context?

For example, let me describe my use case.

I have elderly relatives who have a hard time working with computers in general.
They don't even know where to click on stuff and change preferences.

My idea was to write web-related parts (ruby + sinatra) or even GUI bindings
(ruby + gtk) and to then send emails to me (I have no smartphone so I have
to work with emails here). But dealing with sendmail, smtp etc... all is quite
annoying, so I am looking for one-stack-to-rule-them-all literally. Using
rails is WAY too much setup work for me either; I really want to contain
this as simple as possible. So something like maddy would be fine IF it
would be simple and straightforward to use. For that, documentation is
extremely important - which is ultimately the main reason why I wrote this
issue request. Hopefully you can understand that particular use case.

I in general probe to see whether I can solve a given problem, but only
for a while before I lose interest and move on. This is why the documentation
is so important to me. I need to "get in" and to the most relevant parts
quickly (another reason why StackOverflow was quite a success, since
I can scroll to the more relevant parts ... but this does not work very
well for new software and such).

**Comment by foxcpp** [comment]
There are sure a lot of articles to be added to documentation in addition to reference that is already present.
Topics that I believe should be covered:
- Local account management
- imapsql messages DB inspection
- Diagnostics of inbound/outbound delivery problems (aka "how to read maddy logs")
- Conditional message routing and how it works in maddy

Do you have other topics you think are important to be covered?

**Comment by z3ugma** [comment]
@foxcpp the installation documentation on a VPS without Docker is a bit hard to follow.

Currently on https://maddy.email/tutorials/setting-up/, it says:

```
Your options are:

Pre-built tarball (Linux, amd64)

Available on GitHub or maddy.email.

The tarball contains the full filesystem tree so you can directly extract it into root directory (or you can take just executable).

```

What does the line `The tarball contains the full filesystem tree so you can directly extract it into root directory (or you can take just executable).` mean? Like the tarball should be extracted into the `/` root level of the filesystem and the folders will go the right place? The `build.sh` script stored separately in the source code seems to install a `systemd` agent, `man` pages, move stuff into a `bin` directory, and more. Would you include instructions on how to start from the binary and install on a system?


I'm further running into trouble after building from source. 

```
sudo setfacl -R -m u:maddy:rX /etc/letsencrypt/{live,archive}
```

has you assigning permissions to user `maddy` but there aren't instructions to create a `maddy` user or group in the install instructions.

**Comment by foxcpp** [comment]
I updated https://maddy.email/tutorials/setting-up/ page to be more elaborate about installation from a binary tarball.

**Comment by z3ugma** [comment]
> I updated https://maddy.email/tutorials/setting-up/ page to be more elaborate about installation from a binary tarball.

Thanks, Max! Looks good

**Comment by foxcpp** [comment]
Superseded by separate issues for each topic.
